### ISSUE TITLE
Prebuild doesn't found the specified module and adds "\\?\" to the path

### ISSUE THREAD

**Issue by ndrantotiana** [prebuild/prebuild-install#72]
Hello,

When trying to add [ZeroMQ](https://github.com/zeromq/zeromq.js) module to my Node.js project, I receive:

> The specified module could not be found
\\\\?\\C:\test\node_modules\zeromq\build\Release\zmq.node

The file exists at the path and the issue is occuring when "prebuild-install" tries to require it

> [download.js](https://github.com/prebuild/prebuild-install/blob/master/download.js) Line 134:
`require(resolved)`

it seems that it prepends "\\\\?\\" to the path. Is it normal?

**Comment by mathiask88** [comment]
I don't think `prebuild-install` adds that prefix. Kinda curious that node can't handle those valid windows ([paths](https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247%28v=vs.85%29.aspx)

But I can't reproduce on my machine. Maybe a local thing or a specific node/windows combination?!

**Comment by vweevers** [comment]
I tried a path like that on my machine, `require` does support it.

**Comment by ndrantotiana** [comment]
Have you tried with the last version of ZeroMQ (4.6.0) and with electron 1.8.x (1.8.4 for example)?
I'm using these versions and tried on many Windows machines but haven't get it worked :-(. Any help will be welcome to find if it is "prebuild-install" issue or "zeromq" prebuilt binaries errors.

**Comment by mathiask88** [comment]
Ahh electron.. Seems like https://github.com/prebuild/prebuild-install/issues/68 :)

**Comment by ndrantotiana** [comment]
Yes. Sorry not to have precised it before. What do you think of not to require it if runtime is electron? It may work but is just an easy fix but the process of verification will just be skipped.

The weird thing is that it worked fine with electron 1.7.x but not in electron 1.8.x

**Comment by vweevers** [comment]
@ndrantotiana can you provide a `package.json` (or repo) that we can try `npm install` on?

**Comment by mathiask88** [comment]
> Yes. Sorry not to have precised it before. What do you think of not to require it if runtime is electron? It may work but is just an easy fix but the process of verification will just be skipped.

Yes, it is fine to skip the test. NPM always runs with node, so it is not possible to test/require the electron module. #73 

>The weird thing is that it worked fine with electron 1.7.x but not in electron 1.8.x

The ABI version mismatches here, so the require step was skipped.

**Comment by ndrantotiana** [comment]
@vweevers, I have a monorepo application but you can use the following [package.json](https://gist.github.com/ndrantotiana/a85f163564d945ba580518d35cd18654) file to test it.
After doing a `npm install`, execute the following `npm rebuild zeromq --runtime=electron --target=1.8.4`. It should not try to build ZeroMQ from its source but use the prebuilt version on Windows.
It is using fine the prebuild version if you change the target of the `npm rebuild` from `1.8.4` to `1.7.6`for example.

**Comment by ndrantotiana** [comment]
@mathiask88, sorry to ask you this question but could you assist me on what should I do? Is the fix already at npm repo, should I wait for it to be on npm repo?

**Comment by ralphtheninja** [comment]
@ndrantotiana https://github.com/prebuild/prebuild-install/pull/73 is a fix, but it has to be released first. Sit tight :)

**Comment by ndrantotiana** [comment]
Hope it will be released soon ;).

**Comment by ndrantotiana** [comment]
Thank you :-).
